src/modules/ui/overlay/overlay.ts

```diff
@@ -55,6 +67,10 @@ class Overlay implements IOverlay {
       this.hide();
     }
 
+    if (!config.hideOverlay) {
+      this._mainUIBlock.disableShowingContent();
```

Comment:
why you not hiding loader here?

Reply:
After class Overlay initialized, Overlay's private method _updatePlayingState called and invoked
_tryShowContent() method, in which loader hiding.